src/components/common/CompletingParty.vue

```diff
@@ -1,28 +1,87 @@
 <template>
-  <section>
+  <section id="completing-party-section">
     <header>
       <h2>Original Completing Party</h2>
     </header>
 
-    <div id="completing-party-container">
-      Under construction!
-    </div>
+    <v-card flat class="mt-4">
+      <div class="pa-8">
+        <v-layout row>
+          <v-flex xs3 md2>
+            <label class="font-weight-bold">Legal Name</label>
+            <action-chip v-if="corrected" :actionable-item="{ action: ActionTypes.EDITED }" />
+          </v-flex>
+          <v-flex xs9 md10>
+            <span>{{formatName(completingParty) || "Unknown"}}</span>
+          </v-flex>
+        </v-layout>
+      </div>
+    </v-card>
   </section>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { ActionChip } from '@/components/common'
+import { IncorporationFilingIF, OrgPersonIF } from '@/interfaces'
+import { ActionTypes, Roles } from '@/enums'
 
-@Component({})
+@Component({
+  components: { ActionChip }
+})
 export default class CompletingParty extends Vue {
+  // Declaration for template
+  readonly ActionTypes = ActionTypes
+
+  @Getter getPeopleAndRoles!: Array<OrgPersonIF>
+  @Getter getOriginalIA!: IncorporationFilingIF
+
+  /** The current Completing Party if found, otherwise undefined. */
+  private get completingParty () : OrgPersonIF {
+    return this.getCompletingParty(this.getPeopleAndRoles)
+  }
+
+  /** The original Completing Party if found, otherwise undefined. */
+  private get originalCompletingParty () : OrgPersonIF {
+    return this.getCompletingParty(this.getOriginalIA?.incorporationApplication?.parties)
```

Comment:
Different approach to the getter here! i've no objections at all but curious why this way over another getter for the og completing party?

Reply:
Oh i see.. two local getters using the already existing getters. I'm definitely fine with this!

Reply:
There is a small difference between the getter here and in PeopleAndRoles.vue : here I combined the "get original list" and "get original completing party", partly to shorten the code and partly to include optional chaining.